pkg/controller/servicebindingrequest/servicebinder.go

```diff
@@ -341,6 +341,15 @@ func buildServiceBinder(
 		options.restMapper,
 	)
 
+	if options.sbr.Spec.ApplicationSelector.BindingPath == nil {
+		options.sbr.Spec.ApplicationSelector.BindingPath = &v1alpha1.BindingPath{
+			PodSpecPath: &v1alpha1.PodSpecPath{
+				Containers: defaultPathToContainers,
+				Volumes:    defaultPathToVolumes,
+			},
+		}
+	}
+
```

Comment:
How about that create a function on `ApplicationSelector` that sets various default values:

```go
func (applicationSelector ApplicationSelector) SetDefaults() (ApplicationSelector) {
  if applicationSelector.BindingPath == nill {
    &v1alpha1.BindingPath{
			PodSpecPath: &v1alpha1.PodSpecPath{
				Containers: defaultPathToContainers,
				Volumes:    defaultPathToVolumes,
			},
  }
}
```

you can even declare a default BindingPath instance and use it. With that, we can avoid code duplication and use that function also here: https://github.com/redhat-developer/service-binding-operator/pull/513/files#diff-d422caad0c9d74644056915cb438c96aR104

Perhaps there also other code blocks where we could apply it.

Reply:
+1 on creating a function that sets default values

Reply:
Done. bf98899d20e8c708b86c7b4a6ad26ff0e0e8f216